src/actions/LoginActions.tsx

```diff
@@ -84,6 +84,9 @@ export function initializeAccount(navigation: NavigationBase, account: EdgeAccou
       // during account creation
       await readLocalAccountSettings(account)
 
+      // HACK: Android needs this for some reason for proper logout
+      navigation.pop()
```

Comment:
Would it make more sense to do `navigation.replace` a few lines down? Then we don't need to `pop` the login scene, but can instead `replace` it with the main app.

Also, this way makes a lot of sense. We should search for everywhere we call `navigate('login' | 'gettingStarted' | 'edgeApp')`, and change all those places to `replace`. Then there three scenes become mutually exclusive, and there are no more hacks.